lib/otel/span-processor.js

```diff
@@ -254,4 +272,26 @@ module.exports = class NrSpanProcessor {
     }
     this.#reconciler.reconcile({ segment, otelSpan: span, mapper })
   }
+
+  reconcileLambdaAttributes({ segment, span }) {
+    // see https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/specification/trace/semantic_conventions/faas.md#example
+    // Add entity linking attrs: cloud.resource_id and cloud.platform
+    segment.addAttribute('cloud.platform', 'aws_lambda')
+    const region = span.attributes['cloud.region'] ?? span.attributes['faas.invoked_region'] ?? 'unknown'
+    const accountId = this.agent.config.cloud.aws.account_id ?? 'unknown'
+    const functionName = span.attributes['faas.name'] ?? span.attributees['faas.invoked_name'] ?? 'unknown'
+    segment.addAttribute('cloud.resource_id', `arn:aws:lambda:${region}:${accountId}:function:${functionName}`)
```

Comment:
you also have to set https://github.com/newrelic/node-newrelic/blob/main/lib/instrumentation/aws-sdk/v3/lambda.js#L34C37-L34C45